docs/concepts/ipld.md

```diff
@@ -11,4 +11,53 @@ related:
+
+![A narrow-waist diagram showing where IPLD fits into the internet ecosystem.](https://gateway.ipfs.io/ipfs/QmXgrfpCcSFfXnXqSz6G3V9E21pTZdBmVdsBQCkr86kHXP)
+
+## The IPLD layer model
+
+The IPLD specifications are divided into [layers](https://github.com/ipld/specs#ipld-layer-model):
+
+0. **Block layer:** The block layer contains the content identifier (CID), which is used to self-describe the multi-base-prefix, version, content type, and content address. This takes the guessing out of encoding/decoding.
+1. **Data model layer:** This layer describes what is representable in IPLD (lists, booleans, maps, etc). The kinds of data available are listed in the [`ipld/specs` repository](https://github.com/ipld/specs/blob/master/data-model-layer/data-model.md#kinds).
+1. **Schema layer:** The schema layer uses the data model layer to build more complex layouts.
+
+Below is the simplified hierarchy:
+
+```
+┌──────────────────────────────────────────────────┐
+│                                                  │
+│                   2. Schema layer                │
+│ (advanced types for multi-block data structures) │
+│                                                  │
+├──────────────────────────────────────────────────┤
+│                                                  │
+│                 1. Data model layer              │
+│  (basic types for single-block data structures)  │
+│                                                  │
+├──────────────────────────────────────────────────┤
+│                                                  │
+│                   0. Block layer                 │
+│               (CID, data, codec)                 │
+│                                                  │
+└──────────────────────────────────────────────────┘
+```
+
+## IPLD and "The Merkle Forest"
+
+The heart of IPLD is to unify data models. Links can be traversed across data formats, opening endless possibilities. Imagine having an Ethereum contract referencing data on IPFS, or having a torrent file link to Zcash. Instead of having one large Merkle tree, as with Bitcoin and Ethereum, IPFS uses IPLD to unite the many Merkle trees into what is coined as the Merkle Forest. 🌳🌲🌴
+
+## Further resources
+
+For some examples, you may want to check out the interactive ProtoSchool tutorial [P2P Data Links with Content Addressing](https://proto.school/#/basics/) or learn how to [resolve through IPLD graphs with the dag API](https://github.com/ipfs/js-ipfs/tree/master/examples/traverse-ipld-graphs).
+
+For more information, you may want to check out the [IPLD explorer](https://explore.ipld.io/) and Juan Benet's [Enter the Merkle Forest](https://www.youtube.com/watch?v=Bqs_LzBjQyk&t=2s).
```

Comment:
Should we also link Eric's talk https://media.ccc.de/v/gpn19-105-foundations-for-decentralization-data-with-ipld.